css/style.css

```diff
@@ -1,50 +1,383 @@
-/* CSS files add styling rules to your content */
-
-body {
-  font-family: helvetica, arial, sans-serif;
-  margin: 0;
-}
-
-h1 {
-  font-style: italic;
-  color: #black;
-}
-
-.grid-container {
-  display: grid;
-  grid-template-columns: 200px auto;
-  grid-template-rows: 100px 600px 50px;
-  height: 100vh;
-}
-
-.header {
-  box-sizing: border-box;
-  background-color: red;
-  grid-columns-start: 1;
-  grid-column-end: span 3;
-}
-
-.sidebar {
-  background-color: green;
-}
-
-.content {
-  box-sizing: border-box;
-  background-color: #ffffcc;
-  position: relative;
-  padding-left: 10px;
-  grid-column-end: span 2;
-}
-
-.scroll-area {
-  position: absolute;
-  height: 100%;
-  overflow: auto;
-}
-
-.footer {
-  background-color: yellow;
-  grid-column-end: span 3;
-}
-
-

```

Comment:
All of the CSS that was here before was a placeholder, it now is correct for the layout of the website